code/dqe/attrcheck.q

```diff
@@ -0,0 +1,8 @@
+\d .dqe
+attrcheck:{[tab;attribute;col]
```

Comment:
Add a comment to explain this function.

Reply:
Should the passed attrs + cols be a dict? Would help prevent users passing 2 lists of lengths.